web_src/js/features/emoji.js

```diff
@@ -21,3 +19,16 @@ export const emojiMap = {};
 for (const key of emojiKeys) {
   emojiMap[key] = tempMap[key];
 }
+
+export function emojiHTML(name) {
+  let inner;
+  if (name === 'gitea') {
+    inner = `<img class="emoji" alt=":${name}:" src="${StaticUrlPrefix}/img/emoji.png" align="absmiddle">`;
```

Comment:
why is this emoji.png and not gitea.png?

Reply:
gitea.png could be customized by the user for the site-wide icon and I think we don't want to render the custom image in this case.

Reply:
I guess I wonder why anybody would change gitea.png to be something other than the provided gitea logo or if they did do that for some reason it is probably then what they would want to see here as well.

:gitea: is going to render to gitea.png in the code always so it would be weird to show a different image in the tribute in the off case somebody did make a change to it.

Reply:
Thought I may undo this and just use `gitea.png` as I think that's how it previously worked (using potentially customized image variant).